core-bundle/src/Pow/Altcha/Validator/AltchaValidator.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Pow\Altcha\Validator;
+
+use Contao\CoreBundle\Pow\Altcha\Altcha;
+
+class AltchaValidator
```

Comment:
What is the advantage of this? I think we can just drop this and use the `Altcha` service directly.